toolbox/fdc3-for-web/reference-ui/src/main.ts

```diff
@@ -128,7 +128,7 @@ const openChannelIframe = (e: MouseEvent) => {
   const channel = new MessageChannel();
 
   // STEP 2B: Receive confirmation over port from iframe
-  channel.port1.onmessage = ({ data }) => {
+  channel.port1.addEventListener('message', ({ data }) => {
```

Comment:
Have you tested this code?  I can't get it to run.  If you can't either I suggest we take it out and the index.html file - people will be able to test the channel selector and intent resolver much better through the demo.

Reply:
It looked well out of date to me and out of sync with the UI implementations, I fixed a few things but not tested - it definitely needs work to run.

@julianna-ciq